.github/workflows/isort.yml

```diff
@@ -7,14 +7,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-python@v5
-        with:
-          python-version: 3.9
-      - name: Install apt dependencies
-        # Install `libopenmpi` for mpi4py
-        run: |
-          sudo apt update
-          sudo apt install openmpi-bin libopenmpi-dev
-      # Install dependencies for proper 1st/2nd/3rd party import sorting
```

Comment:
How come this is not required anymore? There will be differences in import sorting if you remove this.

Reply:
Really? I do not understand what these lines would bring as by default this action isort only checks for differences from the previous commit.

Reply:
Well, if we ever add an import in those blocks then they'd be moved to a place they don't belong now.

These lines ensure the following:


```
import os
import sys

from third_party import (lib1, lib2, lib3, lib4, lib5, lib6, lib7, lib8,
                         lib9, lib10, lib11, lib12, lib13, lib14, lib15)

from my_lib import Object, Object2, Object3
```

without it they'd put all the `third_party` sorts together in the `my_lib` sorts. Check the admonition tip on https://pycqa.github.io/isort/docs/configuration/github_action.html